feature-extensions/apigee/jazz-apigee-proxy/jazz-apigee-proxy.js

```diff
@@ -0,0 +1,97 @@
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+const errorHandlerModule = require("./components/error-handler.js");
+const logger = require("./components/logger.js");
+const AWS = require('aws-sdk');
+
+module.exports.handler = (event, context, callback) => {
```

Comment:
@bleggett  You might need a sync with the latest version available here: https://github.com/tmobile/jazz/tree/apigee-plugin/jazz_apigee-proxy-aws

Reply:
As discussed above will remove the code from here and `git clone` as part of the feature extension installer.